ruby_on_rails/apis/project_kittens_api.md

```diff
@@ -21,13 +21,14 @@ In this warmup, you'll get a chance to poke around with an existing API from Fli
     More interestingly, you can see the URL they used to make the request below that. Let's break it apart here to show the parameters more clearly:
 
     ~~~bash
-    http://api.flickr.com/services/rest/
+    https://www.flickr.com/services/rest/
     ?method=flickr.photos.search
-    &api_key=e0eb58bf4b3e29b253e86d6092e69dee
+    &api_key=7b427302f28e47cfbf74da7db99b823b
     &tags=puppies
     &format=json
     &nojsoncallback=1
-    &api_sig=200efb63cb01a3d141fff12585e1e20a
+    &auth_token=72157720893721539-d4bbe5496270bc2f
+    &api_sig=60ced132c8c35ab52607c26eac759023
```

Comment:
I understand why you updated the endpoint on line 24. Unless I am missing something, I don't think we need to change the `api_key` or `api_sig` on lines 26 and 31. If I am missing something, can you please explain it further?

I'm also assuming that `auth_token` is a new addition to the api response and if so, I am good with that addition.

Reply:
Looks like the auth_token was in there because I was logged in. I'll have to remove that. Bear with me as I'm new to this and still going through the curriculum. However If you use the original api_key and api_sig from the lesson you will get a code 100, invalid api key. I can leave them if that doesn't matter or I can generate a new url without a auth_token and add working api_key and api_sig.

Let me know what you think is best!!

Reply:
>  can generate a new url without a auth_token and add working api_key and api_sig

Yes, let's do it this way. Thanks for the explanation and taking the time to help me understand all the changes proposed.